src/ui/components/play-pen-header.ts

```diff
@@ -99,7 +99,7 @@ export class PlayPenHeader extends LitElement {
             appearance="orangered"
             size="small"
             icon="share-new-outline"
-            title="Copy Program URL to Clipboard"
+            title="Copy URL to Clipboard"
```

Comment:
:no_bell: This makes sense to me. Just some background: the original intent here was to indicate that the program / pen is fully represented in the URL which users may not realize. Product banned pen terminology and calling everything a playground is ambiguous.

Reply:
That makes sense. I mainly just want it as short as possible. But also not sure if we've used the term "program" elsewhere in the UI.

As for the overall naming, my mental model is:
Play = Product name
Playground = Product category

Reply:
My thinking is:

- Play: product
- Playground: execution environment or sandbox; there may be multiple of these on a page like the docs and they each have their own independent state.
- Pen: program and the thing you share.